File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        HashMap<Integer,Integer> h=new HashMap<Integer,Integer>();
        for(int i=0;i<nums.length;i++){
            System.out.println(nums[i]+" "+(h.getOrDefault(h.get(nums[i]),0)+1));
            h.put(nums[i],h.getOrDefault(nums[i],0)+1);
        }
        int count=0;
        int i,j;
        for( i=0;i<3;i++){
            if(h.containsKey(i)){
                for( j=count;j<(count+h.get(i));j++){
                    nums[j]=i;
                }
                count=j;
            }
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zerop, twop = 0, len(nums)-1
        pointer = 0
        while pointer<=twop:
            if nums[pointer] == 0:
                nums[zerop], nums[pointer] = nums[pointer], nums[zerop]
                zerop += 1
                pointer += 1
            elif nums[pointer] == 1:
                pointer += 1
            else:
                nums[pointer], nums[twop] = nums[twop], nums[pointer]
                twop -= 1
        